### PR TITLE
Allow a bare /cc command to request a review from commenter.

### PR DESCRIPTION
**Pull request by cjwagner**
/cc @fejta @spxtr

### CODE REVIEW

File: prow/plugins/assign/assign.go [hunk 1]
```diff
@@ -232,14 +226,13 @@ func newAssignHandler(e *event, gc githubClient, log *logrus.Entry) *handler {
 
 func newReviewHandler(e *event, gc githubClient, log *logrus.Entry) *handler {
 	addFailureResponse := func(mu github.MissingUsers) string {
-		return fmt.Sprintf("GitHub didn't allow me to request PR reviews from the following users: %s.\n\nNote that only people with write access to %s/%s can review this PR.\n", strings.Join(mu.Users, ", "), e.org, e.repo)
+		return fmt.Sprintf("GitHub didn't allow me to request PR reviews from the following users: %s.\n\nNote that only people with write access to %s/%s can review this PR and the PR author cannot review their own PR.\n", strings.Join(mu.Users, ", "), e.org, e.repo)
```
Comment (fejta) [inline prow/plugins/assign/assign.go]: nit: s/only people/only other people/ might be a less verbose way to say the same thing

Comment (cjwagner) [inline prow/plugins/assign/assign.go]: done

**Comment by spxtr** [comment]
What happens if I `/cc` on my own PR?

**Comment by fejta** [comment]
It leaves an error message

**Comment by cblecker** [comment]
Please update `commands.md` as well :)
(hopefully will just be making the user arguments optional after https://github.com/kubernetes/test-infra/pull/2636 is merged)

**Comment by cblecker** [comment]
Needs a rebase and hook bump, but LGTM otherwise

**Pull request merged**